static/scss/base/_buttons.scss

```diff
@@ -18,4 +18,14 @@
       width: 100%;
     }
   }
+
+  .button--neutral {
+    background-color: $white;
+    border: 0;
+    color: $text-color;
+
+    &:hover {
+      background-color: darken($white, 30%);
```

Comment:
Can you please follow the Vanilla pattern for this: https://github.com/ubuntudesign/vanilla-design/blob/master/Buttons/buttons.md#neutral